### ISSUE TITLE
Regarding Custom Dataset

### ISSUE THREAD

**Issue by pytholic** [sshaoshuai/PointRCNN#198]
Hi. I have been doing research regarding point cloud object detection. However, all the architectures currently support only benchmark datasets and there is no proper guidance on how to prepare our own datasets. For example, I have some point clouds of patients and I want to detect some regions for medical purposes. However, I cannot find any guidance here or in other models as well to work with custom datasets. Will appreciate any guidance on this. Thanks!

**Comment by Windson9** [comment]
Hi @pytholic. Were you able to train with your custom dataset ?

**Comment by pytholic** [comment]
@Windson9 Not really. I ended up using some segmentation methods. It was long time ago so I do not remember all the details.
Have a look at this <a href="https://github.com/nikitakaraevv/pointnet">repo</a>, it might be useful.

**Comment by OrangeSodahub** [comment]
@Windson9 @pytholic I've implemented the training of the custom dataset, you could refer to [this](https://github.com/OrangeSodahub/CRLFnet/blob/master/src/site_model/src/LidCamFusion/OpenPCDet/pcdet/datasets/custom/README.md)
